testing/postfwd2/postfwd2.initd

```diff
@@ -0,0 +1,31 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+PIDFILE=/run/postfwd2.pid
+
+depend() {
+	need net
+}
+
+start() {
```

Comment:
If possible, could you use the declarative syntax rather than [defining your own start/stop functions][0]?

[0]:https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#dont-write-your-own-startstop-functions